cmd/podman/search.go

```diff
@@ -12,11 +12,6 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const (
-	descriptionTruncLength = 44
```

Comment:
OK, these seem like they ought to be used - did we start hardcoding them when we have constants?